### PR TITLE
Add seed data for Editionable Worldwide Organisations

### PR DESCRIPTION
**Pull request by brucebolt**
This should make it easier for us to test usability in the development environment by giving us some existing seed data.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -196,6 +196,9 @@ en:
       edition_with_appointment:
         one: Edition with appointment
         other: Editions with appointment
+      editionable_worldwide_organisation:
```
Comment (JonathanHallam) [inline config/locales/en.yml:199]: Can you run the rake task so these are in all the locales?

Comment (brucebolt) [reply config/locales/en.yml:199]: Done.  Involved quite a bit of hacking as the rake task doesn't correctly populate the plural rules 😬.

Comment (JonathanHallam) [reply config/locales/en.yml:199]: I'm assuming that's why some are `one, other` and others have a load more numbers?

**Review by JonathanHallam — APPROVED** [review]
👍 thanks

**Pull request merged**